frontend/app/index.html

```diff
@@ -23,7 +23,7 @@
     <link integrity="" rel="stylesheet" href="{{rootURL}}assets/badgeyay.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.1/components/icon.css">
     <link rel="shortcut icon" href="{{rootURL}}favicon.ico">
-    <script src="http://code.jquery.com/jquery-latest.min.js"></script>
+    <script src="https://code.jquery.com/jquery-latest.min.js"></script>
```

Comment:
I suppose that these changes have already been done.
Can you please update your PR with the latest changes?